pack and install test fails

See https://github.com/googleapis/nodejs-spanner/pull/634

Did you start seeing this with #634? Is the error always:

```
 1) 📦 pack and install
       should be able to use the d.ts:
     Error: Command failed: npm install --unsafe-perm
      at Promise.all.then.arr (node_modules/execa/index.js:231:11)
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

Earlier in the build, there's also:

```
> tsc -p .
node_modules/p-queue/index.d.ts:81:54 - error TS2344: Type 'DefaultAddOptions' does not satisfy the constraint 'QueueAddOptions'.
  Index signature is missing in type 'DefaultAddOptions'.
81  EnqueueOptionsType extends PQueue.QueueAddOptions = PQueue.DefaultAddOptions
                                                        ~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/package.json
+++ b/package.json
@@ -67,7 +67,7 @@
     "is": "^3.2.1",
     "lodash.snakecase": "^4.1.1",
     "merge-stream": "^2.0.0",
-    "p-queue": "^5.0.0",
+    "p-queue": "^6.0.2",
     "protobufjs": "^6.8.6",
     "split-array-stream": "^2.0.0",
     "stack-trace": "0.0.10",

--- a/samples/crud.js
+++ b/samples/crud.js
@@ -183,9 +183,7 @@ async function queryData(instanceId, databaseId, projectId) {
     rows.forEach(row => {
       const json = row.toJSON();
       console.log(
-        `SingerId: ${json.SingerId}, AlbumId: ${json.AlbumId}, AlbumTitle: ${
-          json.AlbumTitle
-        }`
+        `SingerId: ${json.SingerId}, AlbumId: ${json.AlbumId}, AlbumTitle: ${json.AlbumTitle}`
       );
     });
   } catch (err) {
@@ -234,9 +232,7 @@ async function readData(instanceId, databaseId, projectId) {
     rows.forEach(row => {
       const json = row.toJSON();
       console.log(
-        `SingerId: ${json.SingerId}, AlbumId: ${json.AlbumId}, AlbumTitle: ${
-          json.AlbumTitle
-        }`
+        `SingerId: ${json.SingerId}, AlbumId: ${json.AlbumId}, AlbumTitle: ${json.AlbumTitle}`
       );
     });
   } catch (err) {

--- a/samples/dml.js
+++ b/samples/dml.js
@@ -387,9 +387,7 @@ async function queryDataWithParameter(instanceId, databaseId, projectId) {
     rows.forEach(row => {
       const json = row.toJSON();
       console.log(
-        `SingerId: ${json.SingerId}, FirstName: ${json.FirstName}, LastName: ${
-          json.LastName
-        }`
+        `SingerId: ${json.SingerId}, FirstName: ${json.FirstName}, LastName: ${json.LastName}`
       );
     });
   } catch (err) {
@@ -620,9 +618,7 @@ async function updateUsingBatchDml(instanceId, databaseId, projectId) {
       const [rowCounts] = await transaction.batchUpdate(dmlStatements);
       await transaction.commit();
       console.log(
-        `Successfully executed ${
-          rowCounts.length
-        } SQL statements using Batch DML.`
+        `Successfully executed ${rowCounts.length} SQL statements using Batch DML.`
       );
     });
   } catch (err) {

--- a/samples/indexing.js
+++ b/samples/indexing.js
@@ -151,9 +151,7 @@ async function queryDataWithIndex(
         ? json.MarketingBudget
         : null; // This value is nullable
       console.log(
-        `AlbumId: ${json.AlbumId}, AlbumTitle: ${
-          json.AlbumTitle
-        }, MarketingBudget: ${marketingBudget}`
+        `AlbumId: ${json.AlbumId}, AlbumTitle: ${json.AlbumTitle}, MarketingBudget: ${marketingBudget}`
       );
     });
   } catch (err) {

--- a/samples/timestamp.js
+++ b/samples/timestamp.js
@@ -157,9 +157,7 @@ async function queryTableWithTimestamp(instanceId, databaseId, projectId) {
     rows.forEach(row => {
       const json = row.toJSON();
       console.log(
-        `SingerId: ${json.SingerId}, VenueId: ${json.VenueId}, EventDate: ${
-          json.EventDate
-        }, Revenue: ${json.Revenue}, LastUpdateTime: ${json.LastUpdateTime}`
+        `SingerId: ${json.SingerId}, VenueId: ${json.VenueId}, EventDate: ${json.EventDate}, Revenue: ${json.Revenue}, LastUpdateTime: ${json.LastUpdateTime}`
       );
     });
   } catch (err) {

--- a/samples/transaction.js
+++ b/samples/transaction.js
@@ -52,9 +52,7 @@ function readOnlyTransaction(instanceId, databaseId, projectId) {
       qOneRows.forEach(row => {
         const json = row.toJSON();
         console.log(
-          `SingerId: ${json.SingerId}, AlbumId: ${json.AlbumId}, AlbumTitle: ${
-            json.AlbumTitle
-          }`
+          `SingerId: ${json.SingerId}, AlbumId: ${json.AlbumId}, AlbumTitle: ${json.AlbumTitle}`
         );
       });
 
@@ -70,9 +68,7 @@ function readOnlyTransaction(instanceId, databaseId, projectId) {
       qTwoRows.forEach(row => {
         const json = row.toJSON();
         console.log(
-          `SingerId: ${json.SingerId}, AlbumId: ${json.AlbumId}, AlbumTitle: ${
-            json.AlbumTitle
-          }`
+          `SingerId: ${json.SingerId}, AlbumId: ${json.AlbumId}, AlbumTitle: ${json.AlbumTitle}`
         );
       });
 

--- a/src/database.ts
+++ b/src/database.ts
@@ -938,18 +938,16 @@ class Database extends ServiceObject {
 
       const snapshot = session.snapshot(options);
 
-      snapshot.begin(
-        (err: null | Error): void => {
-          if (err) {
-            this.pool_.release(session);
-            callback(err);
-            return;
-          }
-
-          this._releaseOnEnd(session, snapshot);
-          callback(err, snapshot);
+      snapshot.begin((err: null | Error): void => {
+        if (err) {
+          this.pool_.release(session);
+          callback(err);
+          return;
         }
-      );
+
+        this._releaseOnEnd(session, snapshot);
+        callback(err, snapshot);
+      });
     });
   }
 

--- a/src/index.ts
+++ b/src/index.ts
@@ -384,9 +384,7 @@ class Spanner extends Service {
       delete reqOpts.instance.nodes;
     }
     if (config.config && config.config.indexOf('/') === -1) {
-      reqOpts.instance.config = `projects/${this.projectId}/instanceConfigs/${
-        config.config
-      }`;
+      reqOpts.instance.config = `projects/${this.projectId}/instanceConfigs/${config.config}`;
     }
     this.request(
       {

--- a/src/session-pool.ts
+++ b/src/session-pool.ts
@@ -16,7 +16,7 @@
 
 import {EventEmitter} from 'events';
 import * as is from 'is';
-import * as PQueue from 'p-queue';
+import PQueue from 'p-queue';
 import trace = require('stack-trace');
 
 import {Database} from './database';

--- a/test/session-pool.ts
+++ b/test/session-pool.ts
@@ -17,7 +17,7 @@
 import * as assert from 'assert';
 import * as events from 'events';
 import * as extend from 'extend';
-import * as PQueue from 'p-queue';
+import PQueue from 'p-queue';
 import * as proxyquire from 'proxyquire';
 import * as sinon from 'sinon';
 import stackTrace = require('stack-trace');

